draft-ietf-quic-transport.md

```diff
@@ -566,8 +566,8 @@ wishes to perform a stateless retry (see {{stateless-retry}}).
 
 A Retry packet (shown in {{retry-format}}) only uses the invariant portion of
 the long packet header {{QUIC-INVARIANTS}}; that is, the fields up to and
-including the Destination and Source Connection ID fields.  The contents of the
-Retry packet are not protected.  Like Version Negotiation, a Retry packet
+including the Destination and Source Connection ID fields.  The Payload of a
```

Comment:
Could go either way, but I'd lean toward not capitalizing this usage, or saying "The Payload field"

Reply:
I realized that we say that it has a Payload here, then say that it doesn't in the next sentence, so I changed this more.